GTG/gtk/data/general_preferences.ui

```diff
@@ -361,7 +361,7 @@
                           <object class="GtkLabel">
                             <property name="visible">True</property>
                             <property name="can_focus">False</property>
-                            <property name="label" translatable="yes">Remove old closed tasks automatically</property>
+                            <property name="label" translatable="yes">Automatically remove old, closed tasks</property>
```

Comment:
I see a potential issue with the comma here... "old closed" is clearer to me than "old, closed" because "old, closed" could be two separate things, if you see what I mean?

Reply:
My thought process on this: "old" and "closed" are coordinate adjectives that equally describe the noun "tasks". The task is both old and closed. Coordinate adjectives are separated by a comma; however, I can remove it if you think that this would impede someone's understanding. I am fine either way :)

Reply:
Yeah, for the avoidance of doubt in the user's mind about whether or not it's an enumeration of independent things, I'd tend to avoid the comma if you don't mind! Because if I was casually reading this I'd wonder "wait so will it delete old tasks and closed tasks?"

Reply:
Removed per below commit. Thanks!